tests/test_parser.py

```diff
@@ -127,6 +133,18 @@ def test_parsing_ox_rdfxml_fallback(self):
         )
         self.assertEqual(len(graph), 6)
 
+    def test_parsing_ox_nquads_bulk_load(self):
+        graph = rdflib.Dataset(store="Oxigraph")
+        graph.parse(_TEST_DIR / "data/test.nq", format="ox-nquads", transactional=False)
+        self.assertEqual(len(graph), 6)
+        self.assertEqual(len(graph.query(_NAMEDGRAPH_QUERY)), 1)
```

Comment:
Instead, it would be great to check the number of triple into the default graph and the named graphs. rdflib behaviors related to named graphs are weird and there might be bad surprises.